quick-start/install-helpers.sh

```
@@ -555,17 +555,109 @@ setup_kind_cluster() {
     
     log_info "Setting up Kind cluster '$cluster_name'..."
     
+    # Ensure container is connected to kind network (critical when running in Docker container)
+    if docker network inspect kind &>/dev/null 2>&1; then
+        local container_id="$(cat /etc/hostname 2>/dev/null || echo "")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)